### PR TITLE
Refactor 'Description Advice' in ConsultationDetails component

### LINKED ISSUE #5196: Discharge: recovered- rename field
**Describe the bug**
The pop-up box for entering the discharge details, when the reason is given as recovered, there is a last input field called "Description advice" the title makes no sense. 

**To Reproduce**

1. Go to the patient file (https://care.coronasafe.in/facility/41c73414-e95e-4e50-b0b9-84bafb3b7440/patient/2d7a2c15-83c3-44b9-9af4-f4d3f791d0c8/consultation/453295d1-37dc-46a5-9b97-c64ef61988fd)
2. Click on 'Discharge from CARE.'
3. Select the reason for discharge as "recovered"
4. See the last field in the pop-up window

**Expected behavior**

- [ ] Change the title of the last field from Description advice to "Discharge SOS Prescription"
- [ ] Change the prescription input module below to an SOS prescription input format

**Screenshots**
Screenshot 2023-03-29 at 7 41 12 PM

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #5196

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Comment by nihal467** [comment]
@aparnacoronasafe 

image


1. When a patient is discharged, the medicines in the consultation form are only copied, and the medicine prescribed during the last log update of the patient are not been copied, is it intentional?
2. Currently, the PRN prescription medicine in consultation form is copied to the discharge prescription in the discharge pop-up similarly, Prescription medication is copied to the discharge SOS Prescription, is it intentional?
3.  the way prescription medications are saved and Discharge SOS prescriptions are saved, using two different templates of recording data, is it intentional?

**Comment by nihal467** [comment]
@Ashesh3 change "Discharge SOS Prescription" to "Discharge PRN Prescription"

File: src/Components/Facility/ConsultationDetails.tsx [hunk 1]
```diff
@@ -539,15 +537,15 @@ export const ConsultationDetails = (props: any) => {
...
+              <div className="my-2">
                 <PRNPrescriptionBuilder
                   prescriptions={PRNAdvice}
                   setPrescriptions={setPRNAdvice}
                 />
               </div>
               <div>
-                <FieldLabel>Description Advice</FieldLabel>
-                <PrescriptionBuilder
+                <FieldLabel>Discharge SOS Prescription</FieldLabel>
```
Comment (nihal467) [inline src/Components/Facility/ConsultationDetails.tsx]:
```suggestion
                <FieldLabel>Discharge PRN Prescription</FieldLabel>
```

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**